File: Java/Car.java
package Java;

public class Car 
{

    private Integer id;
    private String license;
    private Account driver;
    private Integer passengers;

    public Car(String license, Account driver)
    {
        this.license = license;
        this.driver = driver;
    }


    void printDataCar()
    {
        System.out.println("License: " + license + " Driver Name: " + driver.name);
    }


    public Integer getId() {
        return id;
    }


    public void setId(Integer id) {
        this.id = id;
    }


    public String getLicense() {
        return license;
    }


    public void setLicense(String license) {
        this.license = license;
    }


    public Account getDriver() {
        return driver;
    }


    public void setDriver(Account driver) {
        this.driver = driver;
    }


    public Integer getPassengers() {
        return passengers;
    }


    public void setPassengers(Integer passengers) {
        this.passengers = passengers;
    }

    
}

File: PHP/Car.php
<?php

class Car
{
    // Atributes

    private $id;
    private $license;
    private $driver;
    private $passenger;

    // Constructor

    public function __construct($license, $driver)
    {
        $this->license = $license;
        $this->driver  = $driver;
    }

    // Getters & Setters

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getLicense()
    {
        return $this->license;
    }

    public function setLicense($license)
    {
        $this->license = $license;
    }
    
    public function getDriver()
    {
        return $this->driver;
    }

    public function setDriver($driver)
    {
        $this->driver = $driver;
    }

    public function getPassenger()
    {
        return $this->passenger;
    }

    public function setPassenger($passenger)
    {
        $this->passenger = $passenger;
    }

    // Methods 

    public function printDataCar()
    {
        echo 
        "License: " . $this->license .
        " Driver: " . $this->driver->getName() .
        " Document: " . $this->driver->getDocument();
    }
}

?>